src/main/resources/templates/CheckinResponse.ftl

```diff
@@ -0,0 +1,39 @@
+<#import "lib.ftl" as lib>
+10<#t>
+<#-- ok: 1-char, fixed-length required field: 0 or 1 -->
+<@lib.ok value=checkinResponse.ok/>
+<#-- resensitize: 1-char, fixed-length required field: Y or N -->
+<@lib.resensitize value=checkinResponse.resensitize/>
+<#-- magnetic media: 1-char, fixed-length required field: Y or N or U -->
+<@lib.magneticMedia value=checkinResponse.magneticMedia!""/>
```

Comment:
That is true. I use that to handle optional arguments that could be null (that sets null fields to an empty string since null will cause errors when used in a macro. In this case (`magneticMedia`) the empty string means unset so we return `U`, since we need 3 distinct states.